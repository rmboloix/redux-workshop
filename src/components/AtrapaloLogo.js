import React from 'react';

export default () => {
  return (<div className="col-md-4">
    <img src="https://www.atrapalo.com/assets/common/a@2x.png" alt="Atrapalo Tech library logo"/>
    <img src="https://www.atrapalo.com/assets/common/logo-acento@2x.png" alt="Atrapalo Tech library title"/>
  </div>);
};