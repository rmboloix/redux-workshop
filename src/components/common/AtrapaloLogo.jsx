import React from 'react'

const AtrapaloLogo = (props) => (
  <div className='col-md-4'>
    <img height={50}
      src='https://www.atrapalo.com/assets/common/logo-acento@2x.png' alt='Atrapalo Tech library title' />
    <img height={50}
      src='https://www.atrapalo.com/assets/common/a@2x.png' alt='Atrapalo Tech library logo' />
  </div>
)

export default AtrapaloLogo
