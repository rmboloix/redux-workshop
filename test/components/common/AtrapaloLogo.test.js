/* eslint-disable no-undef */
import React from 'react'
import AtrapaloLogo from '../../../src/components/common/AtrapaloLogo'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <AtrapaloLogo />
  ).toJSON()

  expect(tree).toMatchSnapshot()
})
