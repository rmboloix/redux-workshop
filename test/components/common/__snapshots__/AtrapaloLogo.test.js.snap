// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="col-md-4"
>
  <img
    alt="Atrapalo Tech library title"
    height={50}
    src="https://www.atrapalo.com/assets/common/logo-acento@2x.png"
  />
  <img
    alt="Atrapalo Tech library logo"
    height={50}
    src="https://www.atrapalo.com/assets/common/a@2x.png"
  />
</div>
`;
